contracts/managers/payout-manager/IPayoutManager.sol

```diff
@@ -29,9 +29,9 @@ interface IPayoutManager is IVersioned {
     //------------------------
     //  EVENTS
     //------------------------
-    event PayoutCreated(uint256 payoutId, address indexed payoutOwner, IERC20 asset, IERC20 rewardAsset, uint256 totalRewardAmount);
-    event PayoutCanceled(uint256 payoutId, IERC20 asset);
-    event PayoutClaimed(uint256 payoutId, address indexed wallet, uint256 balance, uint256 payoutAmount);
+    event PayoutCreated(uint256 payoutId, address indexed payoutOwner, IERC20 asset, IERC20 rewardAsset, uint256 totalRewardAmount, uint256 timestamp);
```

Comment:
Why add `timestamp`? I think we can extract timestamp/time from event block.

Reply:
True, but in `report-service-eth` we only fetch transaction receipts and event logs which do not contain timestamps themselves. We could still fetch the timestamp by fetching block info since we have the block number, but we don't do it like that anywhere in the report service. All of the relevant events come with timestamps, so it was both easier and more consistent to include timestamp in these events too.